source/Plugins/BattleNetLibrary/BattleNetMetadataProvider.cs

```diff
@@ -27,7 +27,7 @@ public override GameMetadata GetMetadata(Game game)
                 Links = new List<Link>(product.Links)
             };
 
-            gameInfo.Links.Add(new Link("PCGamingWiki", @"http://pcgamingwiki.com/w/index.php?search=" + product.Name));
+            gameInfo.Links.Add(new Link("{DynamicResource LOCBattleNetLinksPCGW}", @"http://pcgamingwiki.com/w/index.php?search=" + product.Name));

```

Comment:
Please do actually test changes you make to see if they work. This will definitely not work and it's not the way you load dynamic resources from the code behind. You need to use `Resources.GetString` method from `IPlayniteAPI` object that's injected into a plugin.

Reply:
Figures, I was thinking it wouldn't be that easy. I think I'll need asistance with this since I don't know exactly how it should be modified. 

For example, from my understanding it should go from this:

`           gameInfo.Links.Add(new Link("{DynamicResource LOCExampleString}", @"http://pcgamingwiki.com/w/index.php?search=" + product.Name));`

To this:
`           gameInfo.Links.Add(new Link((PlayniteApi.Resources.GetString("LOCExampleString")), @"http://pcgamingwiki.com/w/index.php?search=" + product.Name));`

I'll see if I can figure out how to set git and how to build playnite to actually test my PRs and check that they actually work.

Reply:
Yes `PlayniteApi.Resources.GetString` is the right use, but you need to inject PlayniteApi intoBattleNetMetadataProvider first via dependency injection.

Reply:
If it's not too much, can you give me an example how to inject the PlayniteApi? I thought that by having `using Playnite.SDK;` but it looks like I was wrong.